gallery/src/components/NxBinaryDonutChart/NxBinaryDonutChartPage.tsx

```diff
@@ -89,10 +91,11 @@ const NxBinaryDonutChartPage = () =>
             <NxTable.Cell>No</NxTable.Cell>
             <NxTable.Cell>
               The size of the hole in the donut, as a percentage of the donut's overall size.  The default value is 50.
-              The computation and use of this prop ignores the 1px white borders on the inside and outside of the donut.
-              That is, when the component is rendered at its default 30px width, a value of 50 on this prop corresponds
-              to 14px (<NxCode>((30px - (1px * 2)) * 50%)</NxCode>). That 14px is the radius of the inside of the actual
-              donut value arc and background, with the inner border being 1px further inside.
+              The computation and use of this prop ignores the 1px {themingEnabled ? 'dark blue' : 'white'} borders on
```

Comment:
`themingEnabled` doesn't automatically mean dark mode, and I also don't really want our doc text to change between modes.  I'd rephrase this to be accurate in both modes.  Specifically, the border color isn't really important to the point of this sentence, so I'd just remove it.

Reply:
> `themingEnabled` doesn't automatically mean dark mode, and I also don't really want our doc text to change between modes. I'd rephrase this to be accurate in both modes. Specifically, the border color isn't really important to the point of this sentence, so I'd just remove it.

Makes sense: 1a8ce6bb84e87d74dd6ff86528ea40986aad356c